app/Http/Controllers/Controller.php

```diff
@@ -6,14 +6,35 @@
 use Illuminate\Routing\Controller as BaseController;
 use Illuminate\Foundation\Validation\ValidatesRequests;
 use Illuminate\Foundation\Auth\Access\AuthorizesRequests;
+use App\Appointment;
+use App\Client;
+use App\Treatment;
+use App\Http\Resources\AppointmentResource;
+use Illuminate\Http\Request;
 
 class Controller extends BaseController
 {
     use AuthorizesRequests, DispatchesJobs, ValidatesRequests;
-
+    
     public function index()
+    {   
+        return view('index');
```

Comment:
Generally, we tend to not use helpers like this. Can we switch this out for:
```suggestion
        return View::make('index');
```